README.md

```diff
@@ -61,3 +61,9 @@ fn main ()
     safe::puts(c!("Hello, World!"));
 }
 ```
+
+## Crate featurse
+
+- `better-docs` provides more detailed documentation, but requires at least Rust 1.54.
```

Comment:
```suggestion
- `better-docs` provides more detailed documentation, but is unstable (only guaranteed to work for the Rust version that is stable when that specific crate version is released).
```

Reply:
I changed it, thought I used another formulation I felt was clearer